internal/server/handlers.go

```diff
@@ -124,7 +124,14 @@ func (a *API) Signup(c *gin.Context, r *api.SignupRequest) (*api.SignupResponse,
 		return nil, err
 	}
 
-	setAuthCookie(c, bearer, keyExpires)
+	// this cookie is set to send on all infra domains, make it expire quickly to prevent an unexpected org being set on requests to other orgs
```

Comment:
It'd be great to include a little more about why we do this. Something about us wanting to use separate cookies for each org, but since signup happens on a different domain from other requests we need this cookie to login the user after signup.